modules/migrating-to-virt.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * virt/getting_started/virt-getting-started.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="migrating-to-virt_{context}"]
+= Migrating to {VirtProductName}
+
+To migrate virtual machines from an external provider such as VMware vSphere, Red Hat OpenStack, Red Hat Virtualization, or another {VirtProductName} cluster, use the Migration Toolkit for Virtualization (MTV). You can also migrate Open Virtual Appliance (OVA) files created by VMware vSphere.
+
+[NOTE]
+====
+Migration Toolkit for Virtualization is not part of {VirtProductName} and requires separate installation. Because of that, all links in this procedure lead outside of {VirtProductName} documentation.
+====
+
+.Prerequisites
+* The Migration Toolkit for Virtualization Operator link:https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/2.7/html/installing_and_using_the_migration_toolkit_for_virtualization/installing-the-operator_mtv[is installed].
+
+.Procedure
+. link:https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/2.7/html/installing_and_using_the_migration_toolkit_for_virtualization/migrating-vms-web-console_mtv#adding-source-providers[Configure the source provider].
++
+Select VMware vSphere, Red Hat OpenStack, Red Hat Virtualization, OVA files created by VMware vSphere, or another {VirtProductName} cluster as the source provider. Depending on the selection, enter required information such as the URL of the API endpoint or credentials needed to access the virtual machines.
+
+. link:https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/2.7/html/installing_and_using_the_migration_toolkit_for_virtualization/migrating-vms-web-console_mtv#creating-migration-plans-ui[Create a migration plan].
++
+Select from the list of available virtual machines, configure the target provider, and define storage and network mappings.
+
+. link:https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/2.7/html/installing_and_using_the_migration_toolkit_for_virtualization/migrating-vms-web-console_mtv#running-migration-plan_mtv[Run the migration plan].
++
+Watch the progress, view migration logs, or cancel the migration.
+
+. Optional: link:https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/2.7/html/installing_and_using_the_migration_toolkit_for_virtualization/migrating-vms-web-console_mtv#migration-plan-options-ui_mtv[Archive the migration plan].
++
+After confirming that the migration has completed successfully, you can archive the migration plan to delete the logs and clean up all temporary resources created during migration. Once archived, you can permanently delete the migration plan.
```

Comment:
The IBM Style Guide dissuades using "once" to mean "after." 
> Avoid terms that might be misinterpreted. For example, use _once_ to mean _one time_, not to mean _after_ or _when_. Use _since_ in relation to the passing of a period of time, not as a synonym for _because_.

Reference: https://www.ibm.com/docs/en/ibm-style?topic=medium-global-audiences

```suggestion
After confirming that the migration has completed successfully, you can archive the migration plan to delete the logs and clean up all temporary resources created during migration. You can permanently delete an archived migration plan.
```